cfgov/jinja2/v1/_includes/molecules/expandable.html

```diff
@@ -52,10 +63,12 @@
         </div>
     </button>
     <div class="m-expandable_content"
-         {{ 'aria-expanded=true' if settings.is_expanded else '' }}>
+            {{ 'aria-expanded=true' if value.is_expanded else '' }}>
         <div class="m-expandable_content-animated">
-            {{ caller() }}
+            {{ value.paragraph |safe }}
+            {% for item in value.optional_content %}
```

Comment:
`value.optional_content` is not included in code comment documentation starting on L:14